lib/plugins/package/lib/zipService.js

```diff
@@ -19,22 +25,26 @@ module.exports = {
     output.on('open', () => {
       zip.pipe(output);
 
-      this.serverless.utils.walkDirSync(servicePath).forEach((filePath) => {
-        const relativeFilePath = path.relative(servicePath, filePath);
-
-        // ensure we don't include the new zip file in our zip
-        if (relativeFilePath.startsWith('.serverless')) return;
-
-        const shouldBeExcluded =
-          exclude.some(value => relativeFilePath.toLowerCase().indexOf(value.toLowerCase()) > -1);
+      const files = glob.sync('**', {
+        cwd: servicePath,
+        ignore: exclude,
+        dot: true,
+        silent: true,
+      });
```

Comment:
I will take a look at this and amend accordingly if we wish to have the include option back, just drop a note on here to confirm and I will look to implement :)